### DISCUSSION TITLE
How to run this project locally and deploy to cloud?

### DISCUSSION THREAD

**Discussion by emito-k** [supabase/supabase#28571] [Questions]
Hello, I'm trying to run this project on my computer. How can I run it locally on my computer just for testing and then what would be the best way to deploy it to a cloud, I'm planning on using docker for this. I'm not used to next or ran supabase before. I only have experience with React and Angular, and NodeJS backends, this is very new to me. Thank you

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/cli
https://supabase.com/docs/guides/cli/managing-environments
